store/sqlstore/team_store.go

```diff
@@ -1325,21 +1425,15 @@ func (s SqlTeamStore) UserBelongsToTeams(userId string, teamIds []string) (bool,
 }
 
 func (s SqlTeamStore) UpdateMembersRole(teamID string, userIDs []string) *model.AppError {
-	sql := fmt.Sprintf(`
-		UPDATE
-			TeamMembers
-		SET
-			SchemeAdmin = CASE WHEN UserId IN ('%s') THEN
-				TRUE
-			ELSE
-				FALSE
-			END
-		WHERE
-			TeamId = :TeamId
-			AND (SchemeGuest = false OR SchemeGuest IS NULL)
-			AND DeleteAt = 0`, strings.Join(userIDs, "', '"))
-
-	if _, err := s.GetMaster().Exec(sql, map[string]interface{}{"TeamId": teamID}); err != nil {
+	sql, args, err := s.getQueryBuilder().
+		Update("").Table("TeamMembers").
```

Comment:
We can probably just do `Update("TeamMembers")`.

Reply:
Done.